oe_theme.theme

```diff
@@ -840,6 +843,65 @@ function oe_theme_preprocess_paragraph__oe_content_row__variant_inpage_navigatio
   $variables['links'] = $links;
 }
 
+/**
+ * Implements hook_preprocess_paragraph() for paragraph--oe-text-feature-media.html.twig.
+ */
+function oe_theme_preprocess_paragraph__oe_text_feature_media(array &$variables): void {
+  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+  $paragraph = $variables['paragraph'];
+
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+
+  // Bail out if there is no media present.
+  if ($paragraph->get('field_oe_media')->isEmpty()) {
+    return;
```

Comment:
Since here you return early, make sure that at the top you initialize `$variables['image']` as empty since you are assuming that is set with stuff inside the template itself.

Reply:
Done